nilearn/plotting/surface/__init__.py

```diff
@@ -0,0 +1,21 @@
+from nilearn.plotting.surface._utils import (
+    check_surface_plotting_inputs,
+    sanitize_hemi_for_surface_image,
+)
+from nilearn.plotting.surface.surf_plotting import (
+    plot_img_on_surf,
+    plot_surf,
+    plot_surf_contours,
+    plot_surf_roi,
+    plot_surf_stat_map,
+)
+
+__all__ = [
+    "check_surface_plotting_inputs",
+    "plot_img_on_surf",
+    "plot_surf",
+    "plot_surf_contours",
+    "plot_surf_roi",
+    "plot_surf_stat_map",
+    "sanitize_hemi_for_surface_image",
```

Comment:
I want to move `html_surface` under `plotting.surface`. Then we won't need to make `check_surface_plotting_inputs` and `sanitize_hemi_for_surface_image` public. 

Or is it possible that we need those in `nilearn.surface`?

Reply:
if things need to be shared across different modules and be kept private then in this case they should go somewhere in nilearn._utils

Reply:
They are only used by `plotting.html_surface` outside of `plotting.surface`. If I move it under `plotting.surface`, there will be no other module using it.